### PR TITLE
add ocp key work to the job name and remove suffix periodic for perio…

### PR DESCRIPTION
**Pull request by lihongyan1**
…dics will be added automatically when there is cron definition in the job

### CODE REVIEW

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp-e2e-test-harness

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp-414-compat

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-ocp-414-compat

File: ci-operator/config/openshift/insights-operator/openshift-insights-operator-master.yaml [hunk 1]
```diff
@@ -240,6 +248,8 @@ tests:
         name: AWS_REGION_PATH
       - default: /secret/aws/aws-secret-access-key/aws-secret-access-key
         name: AWS_SECRET_ACCESS_KEY_PATH
+      - default: /secret/slack-webhook-url/slack-webhook-url
+        name: SLACK_WEBHOOK_URL
```
[the remark this answers is not in the document]
Comment (lihongyan1) [reply ci-operator/config/openshift/insights-operator/openshift-insights-operator-master.yaml:252]: I will take care of this, these are not my intentional change

File: ci-operator/jobs/rhobs/observability-operator/rhobs-observability-operator-main-periodics.yaml [hunk 2]
```diff
@@ -10,34 +10,24 @@ periodics:
...
-    slack:
-      channel: '#observability-operator-cicd'
-      job_states_to_report:
-      - success
-      - failure
-      - error
-      report_template: '{{if eq .Status.State "success"}} :slack-green: Job *{{.Spec.Job}}*
-        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> {{else}} :failed:
-        Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
-        logs> {{end}}'
```
Comment (jan--f) [inline ci-operator/jobs/rhobs/observability-operator/rhobs-observability-operator-main-periodics.yaml:28]: can you please re-add the reporter config? That has to be done in the generated job config...normally that shouldn't change. See https://docs.ci.openshift.org/docs/how-tos/notification/

Maybe that's why we went with the `periodic` variant initially?

What is the benefit of this change?

Comment (lihongyan1) [reply ci-operator/jobs/rhobs/observability-operator/rhobs-observability-operator-main-periodics.yaml:28]: To integrate with testgrid, ocp the key word is needed

**Pull request closed without merging**